include/chopper/layout/hibf_statistics.hpp

```diff
@@ -136,6 +136,23 @@ class hibf_statistics
                   << "#Total HIBF size no correction:" << to_formatted_BF_size(total_size_no_corr) << "\n\n";
     }
 
+    //!\brief Return the total corrected size of the HIBF in bytes
+    size_t get_total_hibf_memory_size()
```

Comment:
```suggestion
    size_t get_total_hibf_memory_size() const
```

seems like it

Reply:
```suggestion
    size_t total_hibf_size_in_byte()
```

`stats.total_hibf_size_in_byte()` doesn't really need a `get_` prefix to be clear, but it would be nice to know that it is in bytes.

Reply:
This function is not really const, because it might call gather_statistics(). We could make it const, but then we would need something like a finalize() public method that would have to be called after every hierarchical binning on the stats object.

Reply:
Ah, I didn't consider `gather_statistics` being called; then it's fine to be not `const` :)